godot-macros/src/method_registration/mod.rs

```diff
@@ -0,0 +1,137 @@
+/*
+* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, You can obtain one at https://mozilla.org/MPL/2.0/.
+*/
+pub mod register_method;
+pub mod virtual_method_callback;
+
+use proc_macro2::{Ident, TokenStream as TokenStream2};
```

Comment:
Usually we directly import the type:
```
use proc_macro2::TokenStream;
```
The rename as `TokenStream2` is only needed when you simultaneously import `TokenStream`.